setup.py

```diff
@@ -233,15 +252,15 @@ def install_pcl_deps(cfg):
         f"{os.path.join(cfg.VCPKG_DIR, cfg.VCPKG_EXEC)} install --clean-after-build pcl[core,visualization]:{cfg.VCPKG_TRIPLET}")
 
 
-def has_colcon():
-    process = subprocess.Popen("colcon --help", shell=True, stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
+def is_command_available(command):
+    process = subprocess.Popen(f"which {command}", shell=True, stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
```

Comment:
AFAIK, there is no `which` program on Windows. I'd stick with checking the pure command here and adding `--help` option to `colcon` and `git-lfs` commands below.